src/server/src/modules/document.ts

```diff
@@ -33,23 +27,21 @@ export const createDocument = async (
     };
 
     const docName = name + '.pdf';
-    const dateAddedAsDate = new Date(); //today
-    const studentRepository = getRepository(Student);
 
-    const studentToUpdate = await studentRepository.findOne({
+    const studentToUpdate = await Student.findOne({
         where: { id: studentId },
     });
     if (!studentToUpdate) {
         insertResult.message =
-            'Student that uploaded the document does not exist';
+            'Student that uploaded the document no longer exists';
         return insertResult;
     }
 
     const documentToInsert = await Document.create({
         name: docName,
-        docType,
+        type: type,
```

Comment:
can you just pass `type`? or it is necessary to do `type: type`?

Reply:
Wait im not sure what happened here. I didnt see this when i checked after changing the symbol. I’ll change it tmr